NeuralMachineTranslation/BERT/fluid/train/run_with_multi_process.sh

```diff
@@ -0,0 +1,49 @@
+#!bin/bash
+set -xe
+
+cd ./LARK_Paddle_BERT/BERT/
+export CUDA_VISIBLE_DEVICES=0,1,2,3
+
+export FLAGS_cudnn_deterministic=true
+
+export FLAGS_enable_parallel_graph=0
+
+export FLAGS_eager_delete_tensor_gb=0.0
+export FLAGS_fraction_of_gpu_memory_to_use=1.0
+#export FLAGS_memory_fraction_of_eager_deletion=1.0
+
+
+BERT_BASE_PATH=$(pwd)/../../chinese_L-12_H-768_A-12
+TASK_NAME='XNLI'
+DATA_PATH=$(pwd)/../../data
+CKPT_PATH=$(pwd)/../../save
+
+device=${CUDA_VISIBLE_DEVICES//,/ }
+arr=($device)
+num_gpu_devices=${#arr[*]}
+if [ $index = "maxbs" ]; then base_batch_size=78; else base_batch_size=32; fi
+batch_size=`expr ${base_batch_size} \* $num_gpu_devices`
+log_file=${run_log_path}/${model_name}_${task}_${index}_${num_gpu_devices}
+
+python -m paddle.distributed.launch --gpus ${num_gpu_devices}  run_classifier.py --task_name ${TASK_NAME} \
+       --use_cuda true \
+       --do_train true \
+       --do_val false \
+       --do_test false \
```

Comment:
The origin value of `do_val ` and `do_test ` is true, but it will trigger a bug, and this bug will be fixed by @gongweibao next week.